src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -1857,6 +1874,16 @@ public void setReadTimeout(final int timeout) {
     this.readTimeout = timeout;
   }
 
+  /** Set a integrationName for all SDK requests. This method is only for test don't use it */
+  public void setIntegrationName(final IntegrationName integrationName) {
```

Comment:
I think the `IntegrationName` enum is a mistake. 

1. The enum tightly couples the SDK to each integration we've implemented. The SDK should be unaware of downstream projects.
1. The enum leaves no room for users to add their own value. It's not inconceivable that we'd want users to have an opportunity to tell us what they're building with the SDK.

I recommend eliminating the `enum` and accepting a simple `String` instead.